packages/wallet-lib/src/types/Storage/methods/rehydrateState.js

```diff
@@ -8,47 +8,35 @@ const logger = require('../../../logger');
  * Fetch the state from the persistence adapter
  * @return {Promise<void>}
  */
-const rehydrateState = async function rehydrateState() {
+const rehydrateState = async function rehydrateState(walletId) {
```

Comment:
Let's leave this function signature intact to not introduce unnecessary changes to the documentation.
You can access a wallet Id with `this.currentWalletId`